frontend/envs/env.local

```diff
@@ -8,8 +8,8 @@ MSAL_CLIENT_ID=2d77a4a9-b7be-4451-ad47-c151d8b6c05f
 # See https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#redirecturi-considerations
 MSAL_REDIRECT_URI=http://localhost:3000/blank
 MSAL_LOGOUT_URI=http://localhost:3000/
-MSAL_MIN_LOG_LEVEL=INFO
+MSAL_MIN_LOG_LEVEL=WARN
```

Comment:
I'd consider leaving this as is. It might be noisy, but it's also invaluable when things aren't working as expected.